FirebaseAuth/Tests/Unit/PhoneAuthProviderTests.swift

```diff
@@ -208,6 +208,45 @@
       )
     }
 
+    /**
```

Comment:
This is the comment style is from the port from the old Objective-C unit tests.

For new tests, it might be better to use the triple-slash docc format used in the library implementation - https://developer.apple.com/documentation/xcode/writing-symbol-documentation-in-your-source-files